libs/components/ag-grid/src/lib/modules/ag-grid/cell-validator/ag-grid-cell-validator-tooltip.component.ts

```diff
@@ -76,36 +83,39 @@ export class SkyAgGridCellValidatorTooltipComponent {
     this.changeDetector.markForCheck();
   }
 
-  public indicatorShouldShow = true;
   public popoverMessageStream = new Subject<SkyPopoverMessage>();
+
   public validatorMessage: string;
 
   public cellRendererParams: SkyCellRendererValidatorParams;
 
-  constructor(
-    private changeDetector: ChangeDetectorRef,
-    private zone: NgZone
-  ) {}
+  private _hoverTimeout: number;
 
-  public hideIndicator(): void {
-    this.zone.run(() => {
-      this.indicatorShouldShow = false;
-      this.changeDetector.detectChanges();
-    });
-  }
+  constructor(private changeDetector: ChangeDetectorRef) {}
 
   public hidePopover(): void {
+    this.cancelDelayedPopover();
     this.popoverMessageStream.next({ type: SkyPopoverMessageType.Close });
   }
 
-  public showIndicator(): void {
-    this.zone.run(() => {
-      this.indicatorShouldShow = true;
-      this.changeDetector.detectChanges();
-    });
-  }
-
   public showPopover(): void {
+    this.cancelDelayedPopover();
     this.popoverMessageStream.next({ type: SkyPopoverMessageType.Open });
   }
+
+  private scheduleDelayedPopover() {
+    /* istanbul ignore else */
+    if (!this._hoverTimeout) {
+      this._hoverTimeout = window.setTimeout(() => {
+        this.showPopover();
+      }, 300);
```

Comment:
Is this number precise? If we dropped it to `0` would it fail to work? Will `300` _always_ work?

Reply:
`300` was the feature requirement from UX.